### ISSUE TITLE
Ring Version 11 doesn't work, removes all devices from HomeKit 

### ISSUE THREAD

**Issue by databoy01** [dgreif/ring#942]
# Bug Report

### Describe the Bug
 A clear and concise description of the problem...

Once upgraded to version 11, all devices disappear from my HomeKit and until I downgraded to 9.24 stayed that way.  My Hoobs is and was up to date on all other upgrades. If you look at the many reviews on the plug-in page, it seems to be happening to everyone.

### To Reproduce
 - Create an `ApiClient`
 - Attempt to ...
 - See error

### Expected behavior
 A clear and concise description of what you expected to happen.

### Screenshots/Logs
 If applicable, add screenshots and log output to help explain your problem.

### Additional context
 Add any other context about the problem here.

### Environment

 - OS: [e.g. Raspbian]
 - Node.js: [e.g. 12.14.1] 
 - NPM: [e.g 6.13.4] 
 - ring-client-api: [e.g 6.0.3]

Version 4.2.5
Node 16.15
Home ridge Version 1.4.1

**Comment by dgreif** [comment]
cc @mkellsy - Node 16 is required for the latest version.  The details above show node 16.  Any other reason a user might be hitting issues with the update?

**Comment by KevoFL** [comment]
I'm experiencing the same behavior. Tried 11.0.2, same outcome.

**Comment by JSVarga** [comment]
Just rebooted my Synology NAS running Homebridge and the Ring plugin won't load. Error messages:
[5/27/2022, 7:48:29 AM] ====================
[5/27/2022, 7:48:29 AM] ERROR LOADING PLUGIN homebridge-ring:
[5/27/2022, 7:48:29 AM] /homebridge/node_modules/homebridge-ring/lib/homebridge/ring-platform.js:132
        config.cameraStatusPollingSeconds = config.cameraStatusPollingSeconds ?? 20;
                                                                               ^

SyntaxError: Unexpected token '?'
    at wrapSafe (internal/modules/cjs/loader.js:1054:16)
    at Module._compile (internal/modules/cjs/loader.js:1102:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/homebridge/node_modules/homebridge-ring/lib/homebridge/index.js:3:25)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
[5/27/2022, 7:48:29 AM] ====================

Also note that when trying to load the settings through the Homebridge UI the cog just goes round, but if I go directly into the config I can see the Ring platform.

**Comment by JSVarga** [comment]
Fixed mine, it appears to be related to the Node.JS version I think. Anyway I see there is a new native Homebridge for Synology DSM. So I installed that to replace the Docker version, which has all the updates and appears easier to keep updated.
NOTE: I backed up the Docker version of Homebridge and restored to the newly installed native version which worked a treat.
Immediately running new native version and Ring is back. Only anecdotal but appear to be more responsive than the Docker version. Other plug in previous causing issue now working again.
